apps/tlv-app/grails-app/assets/javascripts/menus/search.js

```diff
@@ -264,6 +266,113 @@ function checkAllSensors( labelElement ) {
 
 }
 
+function coverageSearch() {
```

Comment:
This runs 4 times when using 'United States Penitentiary Atlanta, GA, United States' as search criteria.

Reply:
Unable to reproduce when starting from a fresh page load.